ci/taos/checker-pr-gateway.sh

```diff
@@ -205,5 +206,5 @@ echo -e "[DEBUG] Added ${checker_name_list[-1]} module."
 
 # Run all the modules
 echo -e "[DEBUG] Running all modules"
-run_all_checkers checker_cmd_list checker_name_list checker_logfile_list $1 $2 $3 $4 $5 $6
+run_all_checkers checker_cmd_list checker_name_list checker_logfile_list $1 $2 $3 $4 $5 $6 $7
```

Comment:
One question. Where can we see the diff-content information  (e.g., $payload->pull_request->diff_url ) after this PR?